src/parser/jobs/sge/modules/Gauge.tsx

```diff
@@ -177,4 +187,54 @@ export class Gauge extends CoreGauge {
 		}
 	}
 	//#endregion
+
+	private onComplete() {
+		const addersgallLeniency = ADDERSGALL_TIME_REQUIRED / 2
+		const forceGainUtaWindows = this.unableToAct.getWindows().filter(uta => Math.max(0, uta.end - uta.start) >= ADDERSGALL_TIME_REQUIRED)
+		const addersgallExpirationTime = this.addersgallTimer.getExpirationTime(addersgallLeniency, this.parser.currentEpochTimestamp, forceGainUtaWindows, addersgallLeniency)
+		const lostAddersgall = Math.floor(addersgallExpirationTime / ADDERSGALL_TIME_REQUIRED)
+
+		if (lostAddersgall > 0) {
+			this.suggestions.add(new Suggestion({
+				icon: this.data.actions.KERACHOLE.icon,
+				content: <Trans id="sge.gauge.suggestions.lost-addersgall.content">
+					You lost Addersgall due to capping the gauge and letting the timer stop. Your Addersgall actions are your primary healing and mitigation tools, so you should try to use another one before regaining your third stack.
```

Comment:
The WHM copy for their lily gauge is:
```
Use <DataLink action="AFFLATUS_RAPTURE" /> or <DataLink action="AFFLATUS_SOLACE" /> before using other GCD heals. It's okay to cap your lilies if you don't need to heal, move, or weave with them.
```

Should this copy be similar?  Addersgall winds up being part of MP management instead of having a DPS refund, but otherwise it seems similar in function to a lily to me - first choice for GCD heals, but not worth prioritizing running over the cap if you don't need the GCD heal.

Reply:
Sage is arguably in a slightly different position from WHM: Addersgall actions are all oGCDs, and GCD healing gets further punished by not procing Kardia, so I feel like the language as it stands is probably appropriate.